godel_surface_detection/include/godel_surface_detection/services/surface_blending_service.h

```diff
@@ -55,9 +55,13 @@ const static std::string TOOL_NAMESPACE = "process_tool";
 struct ProcessPathDetails
 {
   visualization_msgs::MarkerArray process_boundaries_;
```

Comment:
I think this structure is ugly and unscalable, but I know there's something in the pipe to improve it.